lib/trento/application/usecases/clusters/enriched_cluster.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Trento.EnrichedCluster do
+  @moduledoc """
+  Enriched cluster data
+  """
+
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{}
+
+  @derive {Jason.Encoder, except: [:__meta__, :__struct__]}
+  @primary_key {:cluster_id, :binary_id, autogenerate: false}
+  schema "enriched_clusters" do
```

Comment:
To me an `Enrichedcluster` is the result of a cluster plus information available for enrichment.

What do you think about `clusters_enriching_information`, `clusters_enriching_dataa` or something like this?

Reply:
Renamed to `clusters_enrichment_data`, and the module to `ClusterEnrichmentData`